edx_proctoring/api.py

```diff
@@ -1072,6 +1094,27 @@ def create_exam_attempt(exam_id, user_id, taking_as_proctored=False):
         time_remaining_seconds=time_remaining_seconds,
     )
 
+    # Only create a verified name after the attempt is created, as we need the attempt ID
+    if should_create_verified_name:
+        try:
+            create_verified_name(
+                user,
+                verified_name=full_name,
+                profile_name=profile_name,
+                proctored_exam_attempt_id=attempt.id,
+            )
+        except VerifiedNameEmptyString:
```

Comment:
why try/except rather than checking full_name here?

Reply:
Yeah that sounds like a better way to go